libs/ui-lib-tests/cypress/integration/custom-manifests/2-modifying-existing-custom-manifest.cy.ts

```diff
@@ -0,0 +1,52 @@
+import { commonActions } from '../../views/common';
+
+import { transformBasedOnUIVersion } from '../../support/transformations';
+import { customManifestsPage } from '../../views/customManifestsPage';
+const ACTIVE_NAV_ITEM_CLASS = 'pf-m-current';
+
+describe(`Assisted Installer Custom manifests step`, () => {
+  before(() => {
+    cy.loadAiAPIIntercepts({
+      activeSignal: 'READY_TO_INSTALL',
+      activeScenario: 'AI_CREATE_CUSTOM_MANIFESTS',
+    });
+    transformBasedOnUIVersion();
+  });
+
+  beforeEach(() => {
+    cy.loadAiAPIIntercepts(null);
+    commonActions.visitClusterDetailsPage();
+  });
+  describe('Modifyng existing Custom Manifests', () => {
+    it('Can configure custom manifests step and next button is disabled', () => {
+      commonActions
+        .getWizardStepNav('Custom manifests')
+        .should('have.class', ACTIVE_NAV_ITEM_CLASS);
+      commonActions.getNextButton().should('be.disabled');
+    });
+
+    it('Add valid manifest content', () => {
+      customManifestsPage.getStartFromScratch().click();
+      customManifestsPage.fileUpload().attachFile(`custom-manifests/files/manifest1.yaml`);
+      commonActions.getNextButton().should('be.enabled');
+    });
```

Comment:
This test is failing when I run it.

![image](https://github.com/openshift-assisted/assisted-installer-ui/assets/87187179/93438277-9faf-4955-8460-3b3f30f19c52)